src/components/ModuleCard/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import {
+    Flex,
+    Button,
+    Heading,
+    Spacer,
+    Text,
+    Icon,
+    Stack,
+} from "@chakra-ui/react";
+import { IoTrash } from "react-icons/io5";
```

Comment:
Just want to confirm - is this the same icon as that on Figma? Not that big a case tho